### PR TITLE
Export default jwt.service

### PR DESCRIPTION
**Pull request by matheushf**
The functions getToken, saveToken and destroyToken can now be imported like import { getToken, saveToken, destroyToken } from ".../jwt.service" but also the full object can be imported like import JwtService from ".../jwt.service".

#70

### CODE REVIEW

File: src/components/ArticleActions.vue [hunk 1]
```diff
@@ -9,23 +9,23 @@
     &nbsp;&nbsp;
     <button
       class="btn btn-outline-danger btn-sm"
-      v-on:click="deleteArticle(article.slug)">
+      @click="deleteArticle(article.slug)">
```
Comment (igeligel) [inline src/components/ArticleActions.vue]: I think this was still part of your old PR

File: src/common/jwt.service.js [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-  },
-
-  saveToken(token) {
-    window.localStorage.setItem(ID_TOKEN_KEY, token);
-  },
-
-  destroyToken() {
-    window.localStorage.removeItem(ID_TOKEN_KEY);
-  }
+const getToken = () => {
```
Comment (igeligel) [inline src/common/jwt.service.js]: We would still not be able to import the function as named parameter.

Could you export `getToken` itself too?

It should look similar to:

```js
export const getToken = () => {...}

export default { getToken, ... }
```

You can see an example here on what it should look like: https://github.com/gothinkster/vue-realworld-example-app/blob/master/src/common/config.js

Comment (matheushf) [reply src/common/jwt.service.js]: oh, forgot that, I'll update

File: src/components/ArticleActions.vue (hunk 1 above)
Comment (matheushf) [reply src/components/ArticleActions.vue]: yes, forgot to rebase

Comment (matheushf) [reply src/components/ArticleActions.vue]: oh no, is just that I've created a new branch from the previous, with this commit, so it's actually right

**Pull request merged**